actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -304,19 +344,27 @@ def audio_tag(*sources)
         multiple_sources_tag("audio", sources)
       end
 
+      # Returns an HTML audio tag for the source asset in the public
+      # folder. This uses +audio_tag+ and skips any asset
```

Comment:
You skipped this line in the documentation here: `# lookups by assuming any assets are in the `public` folder.`